src/CodeGen.fs

```diff
@@ -1203,4 +1229,118 @@ type {clientName}(url: string, httpClient: HttpClient) =
+
+        /// <summary>
+        /// Creates {clientName} specifying <see href="T:JsonSerializerOptions">JsonSerializerOptions</see>
+        /// </summary>
+        /// <param name="url">GraphQL endpoint URL</param>
+        /// <remarks>
+        /// In order to enable all F# types serialization and deserealization <b>you must</b> add
+        /// <see href="T:System.Text.Json.Serialization.JsonFSharpConverter">JsonFSharpConverter</see>
+        /// from <a href="https://github.com/Tarmil/FSharp.SystemTextJson">FSharp.SystemTextJson</a> NuGet package yourself
+        /// </remarks>
+        new: url: string * options: JsonSerializerOptions -> SpotifyGraphqlClient
+
+        /// <summary>
+        /// Creates {clientName} specifying <see href="T:HttpClient">HttpClient</see> instance
+        /// </summary>
+        /// <param name="url">GraphQL endpoint URL</param>
+        /// <remarks>
+        /// In order to enable all F# types serialization and deserealization
+        /// <see href="T:System.Text.Json.Serialization.JsonFSharpConverter">JsonFSharpConverter</see> by default is added
+        /// from <a href="https://github.com/Tarmil/FSharp.SystemTextJson">FSharp.SystemTextJson</a> NuGet package
+        /// </remarks>
+        new: url: string * client: HttpClient -> SpotifyGraphqlClient
+
+        /// <summary>Creates {clientName}</summary>
+        /// <param name="url">GraphQL endpoint URL</param>
+        /// <remarks>
+        /// In order to enable all F# types serialization and deserealization
+        /// <see href="T:System.Text.Json.Serialization.JsonFSharpConverter">JsonFSharpConverter</see> by default is added
+        /// from <a href="https://github.com/Tarmil/FSharp.SystemTextJson">FSharp.SystemTextJson</a> NuGet package
+        /// </remarks>
+        new: url: string -> SpotifyGraphqlClient
+    end
+"""
+
+let private sampleFSharpNewtonsoftGraphqlClientFsi projectName clientName =
+        $"""namespace {projectName}
+
+open Newtonsoft.Json
+open Newtonsoft.Json.Linq
+open Fable.Remoting.Json
+open System.Net.Http
+open System.Text
+
+type {clientName} =
+    class
+        /// <summary>Creates {clientName} specifying <see href="T:HttpClient">HttpClient</see> instance</summary>
+        /// <remarks>
+        /// In order to enable all F# types serialization and deserealization
+        /// <see href="T:Fable.Remoting.Json.FableJsonConverter">FableJsonConverter</see> is added
+        /// from <a href="https://github.com/Zaid-Ajaj/Fable.SimpleJson">Fable.SimpleJson</a> NuGet package
+        /// </remarks>
+        /// <param name="url">GraphQL endpoint URL</param>
+        new: url: string * client: HttpClient -> SpotifyGraphqlClient
+
+        /// <summary>Creates {clientName}</summary>
+        /// <remarks>
+        /// In order to enable all F# types serialization and deserealization
+        /// <see href="T:Fable.SimpleJson.FableJsonConverter">FableJsonConverter</see> is added
```

Comment:
This documentation is wrong, the package `Fable.SimpleJson` is Fable-only and doesn't have `FableJsonConverter`

Reply:
@Zaid-Ajaj, could you suggest the right XML comment here?

Reply:
I suggest to remove the XML remarks for now and only leave `<summary>Creates {clientName}</summary>`